docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
     depends_on:
       - postgres
     volumes:
+      - ./:/code/
```

Comment:
I think @copelco has tried to avoid pulling in the whole folder like this. In the past, we've added files in the root dir, like frontend config files, by explicitly listing them

Reply:
Ah yes, it does appear to be double mounting currently. @georgehelman What files were not auto-updating with the existing mounts below?

Reply:
If the full tree is mounted, then these files will not be available in the dev container:

https://github.com/deardurham/dear-petition/blob/833fa8fac35e251f5bf7ba8f0d442540ef1d8c95/Dockerfile#L92